server/package.json

```diff
@@ -7,7 +7,7 @@
     "express": "^4.16.3",
     "formidable": "*",
     "node-tesseract": "*",
-    "watson-developer-cloud": "3.8.0"
+    "ibm-watson": "*"
```

Comment:
I think we should also pin the `ibm-watson` version rather than getting latest.

Reply:
@sanjeevghimire  Sure, I have updated the version to 4.1.0 now.